### ISSUE TITLE
Design standards for collectionsCard

### ISSUE THREAD

**Issue by rafaelcorreiapoli** [envisioning/tdb#244]
Despite of different the inner content, I think that these cards should have the same panel actions design (edit, delete, insert new subcollection, copy, color (white or gray) etc...)
![padronizar1](https://cloud.githubusercontent.com/assets/12840136/15115966/c5ee0030-15d7-11e6-9550-48eff945b29f.png)
![padronizar2](https://cloud.githubusercontent.com/assets/12840136/15115964/c5ea7ea6-15d7-11e6-800b-bca33a4ca51c.png)

What should we go for ?

**Comment by arthursoares** [comment]
sorry, I didn't get this. Could you be more specific?

**Comment by rafaelcorreiapoli** [comment]
![diferencas](https://cloud.githubusercontent.com/assets/12840136/15252738/b804ec1e-1905-11e6-80a9-32e4eb25177a.png)

Repare nestes dois componentes, ambos deveriam ser o mesmo (eles representam uma collection), com a única diferença que um mostra as subcollections de forma resumida e outro as mostra de forma completa.
Devemos unificar a disposição dos botões, cor do header/footer do box e funcionalidades
PS: no componente 2 (o branco) dentro da engrenagem tem um dropdown com uma ação importante: adicionar uma nova subcollection

Devemos chegar em um modelo único para facilitar o entendimento do usuário e para reaproveitarmos componentes

**Comment by arthursoares** [comment]
Concordo contigo! Vamos fazer isso. Coloca o dropdown com as mesmas funções!
